command/console.go

```diff
@@ -30,16 +31,39 @@ type ConsoleCommand struct {
 }
 
 func (c *ConsoleCommand) Run(args []string) int {
+	ctx := context.Background()
```

Comment:
is this one too fast for handleTermInterrupt(c.Ui)?

Reply:
The console command uses a REPL system that handles signals on its own, so for now I let it be simple.